src/v2/Apps/Artists/Routes/ArtistsByLetter.tsx

```diff
@@ -0,0 +1,142 @@
+import { Box, Text, media, space } from "@artsy/palette"
+import React, { useState } from "react"
+import { graphql, createRefetchContainer, RelayRefetchProp } from "react-relay"
+import styled, { css } from "styled-components"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { useRouter } from "v2/Artsy/Router/useRouter"
+import { PaginationFragmentContainer } from "v2/Components/Pagination"
+import { ArtistsTopNav } from "../Components/ArtistsTopNav"
+import { ArtistsByLetter_viewer } from "v2/__generated__/ArtistsByLetter_viewer.graphql"
+import { ArtistsByLetterMeta } from "../Components/ArtistsByLetterMeta"
+
+const Columns = styled(Box)<{ isLoading: boolean }>`
+  column-count: 4;
+  ${media.md`column-count: 3;`}
+  ${media.xs`column-count: 1;`}
+
+  ${({ isLoading }) =>
+    isLoading &&
+    css`
+      opacity: 0.25;
+      pointer-events: none;
+    `}
+`
+
+const Name = styled(RouterLink)`
+  display: block;
+  text-decoration: none;
+  padding: ${space(0.5)}px 0;
+`
+
+interface ArtistsByLetterProps {
+  viewer: ArtistsByLetter_viewer
+  relay: RelayRefetchProp
+}
+
+export const ArtistsByLetter: React.FC<ArtistsByLetterProps> = ({
+  relay,
+  viewer: {
+    artistsConnection: {
+      artists,
```

Comment:
(nonblocking) because of our null data issues, i've noticed multi-level destructuring in a function's arguments often have to be unwound with `foo?.bar?.baz` due to blow-ups, or something like:

```tsx
if (!viewer.artistsConnection?.artists) {
  return null
}
```
not too ideal, but def a practical concern given MP's limitations. If yr certain things are all good then np tho!